configs/RC405/config.h

```diff
@@ -0,0 +1,208 @@
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        RC405
+#define MANUFACTURER_ID   RADIOCOMP
+
+#define USE_ACC
+#define USE_GYRO
+
+#define USE_ACC_SPI_MPU6000          // For MPU6000 variation
+#define USE_GYRO_SPI_MPU6000         // For MPU6000 variation
+
+#define USE_ACC_SPI_ICM42688P        // For ICM42688P variation
+#define USE_GYRO_SPI_ICM42688P       // For ICM42688P variation
+
+#define USE_ACCGYRO_BMI270           // For BMI270 variation
+
+#define USE_BARO
+#define USE_BARO_BMP280
+
+#define USE_MAG
+#define USE_MAG_QMC5883
+
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25Q128FV
+#define USE_FLASH_W25N01G
+#define USE_MAX7456
+
+#define SYSTEM_HSE_MHZ 8
+
+#ifndef USE_CAMERA_CONTROL
+#define USE_CAMERA_CONTROL
+#endif
+
+/**********************************************************************
+ * Motor                                                              *
+ **********************************************************************/
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PC8
+#define MOTOR4_PIN           PC9
+
+/**********************************************************************
+ * SERVO                                                              *
+ **********************************************************************/
+#define SERVO1_PIN           PA9
+#define SERVO2_PIN           PA10
+
+#define RX_PPM_PIN           PC7  
+
+/**********************************************************************
+ * UART Bus                                                           *
+ **********************************************************************/
+#define UART1_TX_PIN         PB6
+#define UART1_RX_PIN         PB7
+
+#define UART2_TX_PIN         PA2
+#define UART2_RX_PIN         PA3
+
+#define UART3_TX_PIN         PB10
+#define UART3_RX_PIN         PB11
+
+#define UART4_TX_PIN         PA0
+#define UART4_RX_PIN         PA1
+
+#define UART5_TX_PIN         PC12
+#define UART5_RX_PIN         PD2
+
+#define UART6_TX_PIN         PC6
+#define UART6_RX_PIN         PC7
+
+/**********************************************************************
+ * I2C Bus                                                            *
+ **********************************************************************/
+#define USE_I2C
+#define USE_I2C_DEVICE_1
+
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define MAG_I2C_INSTANCE  (I2CDEV_1)
+
+/**********************************************************************
+ * LED & LED strip & Buzzer                                           *
+ **********************************************************************/
+#define LED0_PIN             PC15
+#define LED1_PIN             PC14
```

Comment:
PC13 - PC15 should no be used for LED

See https://betaflight.com/docs/development/manufacturer/manufacturer-design-guidelines#3146-leds